test/Display/displayMock.test.js

```diff
@@ -0,0 +1,24 @@
+import { Display } from '../../lib/Display/display'
```

Comment:
Mocks are never tested! They are testing tools, but should be very simple. We can go over this more in the pairing session tomorrow! For now, delete this file; it's not needed!

Reply:
This file is now deleted. :)